apps/src/templates/instructions/RubricField.jsx

```diff
@@ -85,6 +84,7 @@ class RubricField extends Component {
     return (
       <div style={styles.rubricLevelHeaders}>
         <div
+          id="unit-test-rubric-field"
```

Comment:
It's generally best to avoid mixing test and product code. If you find yourself adding elements that will only be used for tests, it usually means either the product code or the test code needs to be refactored. I'll add some ideas for how to achieve this without the "unit-test" id in a few mins.

Reply:
Thanks for chiming in, Jessica. I saw you linked to another PR, but can you be more specific about what alternate pattern you're recommending here? We do sometimes use element ids that are only needed for testing, but are always open to new ways to do things.

Unless I missed something, it looks like the new `unit-test-rubric-field` id is not being used anywhere, so in this particular case the `unit-test-...` should probably just be removed.

Reply:
Sorry for the confusion! The link was for a different conversation (regarding chai-enzyme). I'll add more details regarding the link in that thread.

Regarding unit-test-rubric-field: You are right - it looks like it's this isn't used anywhere and can just be removed.